### PR TITLE
Front-end update

### PR DESCRIPTION
**Pull request by mathias-github**
- Updated README file with useful information for new Cuberite users
- Better-looking server icon
- Removed broken wiki link

### CODE REVIEW

**Comment by madmaxoft** [comment]
I have to say I like the old favicon more than the new one. How about a compromise, make the white cube a bit larger, something in between the two versions?

File: Server/README.txt [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-| Founder: Kevin Bansberg     |
-|       A.K.A. FakeTruth      |
-| Lead dev: Mattes D          |
-|       A.K.A. _Xoft(o)       |
-|       A.K.A. xoft           |
-| Monsters by Alex Sonek      |
-|       A.K.A. Duralex        |
-|=============================|
-| Info: www.cuberite.org      |
-\=============================/
```
Comment (tigerw) [inline Server/README.txt]: Nooooo! Not the panel since the beginning of the project! :)

Comment (NiLSPACE) [inline Server/README.txt]: :(
pic

**Comment by mathias-github** [comment]
How about we keep the panel and add the extra info below it?

**Comment by mathias-github** [comment]
@madmaxoft How is this icon? http://i.imgur.com/erMWGI7.png

**Comment by bearbin** [comment]
Nah, get rid of the panel. We like change :P

I think the new text is good.

**Comment by madmaxoft** [comment]
@mathias-github that one looks best.

**Comment by tigerw** [comment]
> Nah, get rid of the panel. We like change :P

vice versa

**Comment by mathias-github** [comment]
Ready to be merged if you want to merge it.

**Comment by SphinxC0re** [comment]
Please rebase and squash your commits onto the current master

**Comment by mathias-github** [comment]
Alright, this PR should be ready now.

**Pull request merged**